Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Placeholder.cs

```diff
@@ -52,6 +50,30 @@ public ImageSource PlaceholderSource
             set { SetValue(PlaceholderSourceProperty, value); }
         }
 
+        private static void PlaceholderSourceChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var control = d as ImageExBase;
+
+            if (e.OldValue == null || e.NewValue == null || !e.OldValue.Equals(e.NewValue))
+            {
+                RoundImageEx round = control as RoundImageEx;
```

Comment:
Discussion: There should be a virtual OnPlaceholderSourceChanged method that RoundImageEx can override and all this code should go in there, rather than be in the base class.